src/main/resources/com/devonfw/tools/solicitor/sql/sources_tobeincluded.sql

```diff
@@ -2,7 +2,6 @@
 --
 -- returns data needed to create script for downloading all sources which need to be included in the distribution
 select distinct
-	REGEXP_REPLACE(a."applicationName",'\s','') as "applicationName",
```

Comment:
Please do not change the existing SQL/report as this might break things that users might rely on (e.g. if they use the generated script to delivery the sources). Create a new version of the sql (e.g. sources_for_scancode.sql) which include these modifications.